app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= image_tag "owl_media_ranker.jpg" %>
```

Comment:
To make this page responsive, you'll need to use a section for each type of media, with using the [Foundation grid layout](https://www.sitepoint.com/foundation-6-the-new-flex-grid/).